### PR TITLE
Changed Topology view to Graph view as text wherever required.

### PR DESCRIPTION
**Pull request by rishumehra**
Changed text "**Topology view**" to "**Graph view**" in the documentation.

For Openshift 4.7

This change is based on the dev issue: https://issues.redhat.com/browse/ODC-5355

### CODE REVIEW

File: modules/odc-interacting-with-applications-and-components.adoc [hunk 1]
```diff
@@ -5,9 +5,9 @@
 [id="odc-interacting-with-applications-and-components_{context}"]
 = Interacting with applications and components
 
-The *Topology view* in the *Developer* perspective of the web console provides the following options to interact with applications and components:
+The *Topology* view in the *Developer* perspective of the web console provides the following options to interact with applications and components:
```
Comment (gajanan-more) [inline modules/odc-interacting-with-applications-and-components.adoc:8]:
```suggestion
The *Topology* page in the *Developer* perspective of the web console provides the following options to interact with applications and components:
```

File: modules/odc-interacting-with-applications-and-components.adoc [hunk 2]
```diff
@@ -37,5 +37,5 @@ The application resource name is appended with indicators for the different type
 +
 [NOTE]
 ====
-Serverless applications take some time to load and display on the *Topology view*. When you create a serverless application, it first creates a service resource and then a revision. After that, it is deployed and displayed on the *Topology view*. If it is the only workload, you might be redirected to the *Add* page. After the revision is deployed, the serverless application is displayed on the *Topology view*.
+Serverless applications take some time to load and display on the *Graph view*. When you create a serverless application, it first creates a service resource and then a revision. After that, it is deployed and displayed on the *Graph view*. If it is the only workload, you might be redirected to the *Add* page. After the revision is deployed, the serverless application is displayed on the *Graph view*.
```
Comment (gajanan-more) [inline modules/odc-interacting-with-applications-and-components.adoc:40]:
```suggestion
Serverless applications take some time to load and display on the *Graph view*. When you deploy a serverless application, it first creates a service resource and then a revision. After that, it is deployed and displayed on the *Graph view*. If it is the only workload, you might be redirected to the *Add* page. After the revision is deployed, the serverless application is displayed on the *Graph view*.
```

File: modules/odc-viewing-application-topology.adoc [hunk 3]
```diff
@@ -5,9 +5,9 @@
 [id="odc-viewing-application-topology_{context}"]
 = Viewing the topology of your application
 
-You can navigate to the *Topology view* using the left navigation panel in the *Developer* perspective. After you create an application, you are directed automatically to the *Topology view* where you can see the status of the application pods, quickly access the application on a public URL, access the source code to modify it, and see the status of your last build. You can zoom in and out to see more details for a particular application.
+You can navigate to the *Topology* view using the left navigation panel in the *Developer* perspective. After you create an application, you are directed automatically to the *Graph view* where you can see the status of the application pods, quickly access the application on a public URL, access the source code to modify it, and see the status of your last build. You can zoom in and out to see more details for a particular application.
```
Comment (gajanan-more) [inline modules/odc-viewing-application-topology.adoc:8]:
```suggestion
You can navigate to the *Topology* page using the left navigation panel in the *Developer* perspective. After you deploy an application, you are directed automatically to the *Graph view* where you can see the status of the application pods, quickly access the application on a public URL, access the source code to modify it, and see the status of your last build. You can zoom in and out to see more details for a particular application.
```

File: modules/odc-viewing-application-topology.adoc [hunk 4]
```diff
@@ -5,9 +5,9 @@
...
 
-The *Topology view* also provides you the option to monitor your applications using the *List* view. Use the *List view* icon (image:odc_list_view_icon.png[title="List view icon"]) to see a list of all your applications and use the *Topology view* icon (image:odc_topology_view_icon.png[title="Topology view icon"]) to switch back to the graph view.
+The *Topology* view also provides you the option to monitor your applications using the *List* view. Use the *List view* icon (image:odc_list_view_icon.png[title="List view icon"]) to see a list of all your applications and use the *Graph view* icon (image:odc_topology_view_icon.png[title="Topology view icon"]) to switch back to the graph view.
```
Comment (gajanan-more) [inline modules/odc-viewing-application-topology.adoc:10]:
```suggestion
The *Topology* page also provides you the option to monitor your applications using the *List* view. Use the *List view* icon (image:odc_list_view_icon.png[title="List view icon"]) to see a list of all your applications and use the *Graph view* icon (image:odc_topology_view_icon.png[title="Topology view icon"]) to switch back to the graph view.
```

**Review by gajanan-more — CHANGES_REQUESTED** [review]

File: modules/odc-interacting-with-applications-and-components.adoc [hunk 5]
```diff
@@ -5,9 +5,9 @@
 [id="odc-interacting-with-applications-and-components_{context}"]
 = Interacting with applications and components
 
-The *Topology view* in the *Developer* perspective of the web console provides the following options to interact with applications and components:
+The *Topology* view in the *Developer* perspective of the web console provides the following options to interact with applications and components:
 
-* Use the *Shortcuts* menu on the upper-right of the screen to navigate components in the *Topology view*.
+* Use the *Shortcuts* menu on the upper-right of the screen to navigate components in the *Graph view*.
```
Comment (andrewballantyne) [inline modules/odc-interacting-with-applications-and-components.adoc:10]: Hmmm... the Shortcuts menu is NEXT to the icon, it's not the name of the icon. it's a different interaction point.

Maybe something like:
```suggestion
* Use the icon (<insert the icon here>) in the upper-right of the screen to navigate components in the *Graph view*.
```

**Review by andrewballantyne — CHANGES_REQUESTED** [review]

**Comment by rishumehra** [comment]
@andrewballantyne As discussed with you over slack I have dropped the Shortcuts menu bullet point as it is not marking any major functionality.

@gajanan-more After having a discussion with @Preeticp we've finalized to retain the "**Topology** view" as is. Otherwise, we need to make the same change to the whole documentation.

@andrewballantyne @gajanan-more @Preeticp Kindly check the changes in the commit. If no further changes are required then please merge this PR.

**Review by andrewballantyne — APPROVED** [review]
👍  LGTM

**Comment by ahardin-rh** [comment]
LGTM! We just need to squash your commits down to one and we should be good to go. Thank you!

**Comment by rishumehra** [comment]
> LGTM! We just need to squash your commits down to one and we should be good to go. Thank you!

@ahardin-rh Squashed commits now. Please go ahead and merge the PR now. Thanks for the review.

**Pull request merged**